pkg/updatestatus/nodeinformer.go

```diff
@@ -155,6 +165,52 @@ func (c *nodeInformerController) sync(ctx context.Context, syncCtx factory.SyncC
 	return nil
 }
 
+type machineConfigVersionCache struct {
+	cache map[string]string
+	lock  sync.Mutex
+}
+
+func (c *machineConfigVersionCache) ingest(mc *machineconfigv1.MachineConfig) bool {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+
+	v, ok := c.cache[mc.Name]
+	if openshiftVersion, exist := mc.Annotations[mco.ReleaseImageVersionAnnotationKey]; exist && openshiftVersion != "" {
```

Comment:
nit: I think we could benefit from some naming symmetry here, like:

```
cachedVersion, cached := c.cache[name]
mcVersion, annotated := mc.Annotations[mco.ReleaseImageVersionAnnotationKey]
if annotated && mcVersion != "" {
  if !cached || cachedVersion != mcVersion {
  ...
  }
} else if cached {
}
````

Reply:
Done.
https://github.com/openshift/cluster-version-operator/pull/1164/commits/a81ec69e31f504ce10a44650cfa5c4d4260fd2cd#diff-c5a41cb83200e13544e0509a5e77c180e16a1d70d7411a79cac2bc107e89950fR178